pages/wikilist/components/UserList/UserCard/UserLink/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import styles from './styles.module.scss';
+import LinkIcon from '@/assets/icons/ic_link.svg';
+import Image from 'next/image';
+
+interface UserLinkProps {
+  url: string;
+}
+
+const UserLink: React.FC<UserLinkProps> = ({ url }) => {
+  return (
+    <div className={styles.linkContainer}>
+      <Image src={LinkIcon} alt="Link icon" width={20} height={20} />
+      <a href={url} target="_blank" rel="noopener noreferrer" className={styles.linkText}>
+        {url}
+      </a>
+    </div>
```

Comment:
```suggestion
import Link from 'next/link';
...
생략
...
<div className={styles['link-container']}>
      <Image src={LinkIcon} alt="링크" width={20} height={20} />
      <Link href={url} target="_blank" rel="noopener noreferrer" className={styles['link-text']}>
        {url}
      </Link>
    </div>
```
1. Next.js에서는 `<a>` 대신 `<Link>`를 사용합니다!
2. `className`은 kebab으로 변경해주세요!
3. `alt`값은 한국어로 직관적으로 수정해주세요!

나머지 부분도 동일하게 바꿔주시면 좋을 것 같아요 ㅎㅎ